plugins/contextmenu/plugin.js

```diff
@@ -140,8 +140,22 @@ CKEDITOR.plugins.add( 'contextmenu', {
 		} );
 
 		editor.addCommand( 'contextMenu', {
-			exec: function() {
-				editor.contextMenu.open( editor.document.getBody() );
+			exec: function( editor, data ) {
+				var offsetX = 0,
+					offsetY = 0,
+					rect;
+
+				// #1451 when opening context menu via keystroke there is no offsetX and Y passed.
+				if ( data.from === 'keystrokeHandler' ) {
```

Comment:
Is this command used in non-keyboard situations? If not, then this conditional seems unnecessary.

Probably it could be used also in programmatic way, but the question is: what are drawbacks of disabling positioning menu in such situation?